lms/djangoapps/courseware/views/views.py

```diff
@@ -857,6 +862,7 @@ def course_about(request, course_id):
             'sneakpeek_allowed': sneakpeek_allowed,
             'course_image_urls': overview.image_urls,
             'reviews_fragment_view': reviews_fragment_view,
+            'banner_account_activation_message': banner_account_activation_message,
```

Comment:
is there a reason why this couldn't be: 
```python
'banner_account_activation_message': get_banner_account_activation_message(request.user),
```

This would also remove 4 lines of forking...